test/e2e/ingress.go

```diff
@@ -62,6 +63,9 @@ var (
 	expectedLBCreationTime    = 7 * time.Minute
 	expectedLBHealthCheckTime = 7 * time.Minute
 
+	// Labels applied throughout, to the RC, the default backend, as a selector etc.
+	controllerLabels = map[string]string{"name": "glbc"}
+
 	// Name of the loadbalancer controller within the cluster addon
 	lbContainerName = "l7-lb-controller"
```

Comment:
Both the controller and the pod have the same name, because there's an insipid sidecar container in there that's not worth including in the controller name.